### DISCUSSION TITLE
Does LAMMPS-SevenNet restrat work?

### DISCUSSION THREAD

**Discussion by turbosonics** [MDIL-SNU/SevenNet#109] [Q&A]
From LAMMPS log file, I got following message: 
> WARNING: Pair style restartinfo set but has no restart support (src/pair.cpp:754)

SevenNet LAMMPS MD simulation still creates restart file. But after reading that message, I'm not sure if the MD simulation that I restarted using restart file accurate or not.  

Devs, do you recommend using LAMMPS restart file to restart LAMMPS MD? Or better restart from geometry, not from restart file?

**Comment by YutackPark** [comment]
@gasplant64 , do you have any idea?
